### ISSUE TITLE
Is there any update plan after 2.2.1?

### ISSUE THREAD

**Issue by chanmaoinc** [bdlukaa/fluent_ui#59]
Look forward to the new update, thanks.

**Comment by bdlukaa** [comment]
Hey! I have been really busy with some stuff, but yes, version 3.0.0 is in the plans.

While the version is not published, you can use the GitHub version: 

```dart
dependencies:
  fluent_ui:
    git: https://github.com/bdlukaa/fluent_ui.git
```

**Comment by chanmaoinc** [comment]
Very happy to hear that. Thanks for you reply and look forward.

> Hey! I have been really busy with some stuff, but yes, version 3.0.0 is in the plans.
> 
> While the version is not published, you can use the GitHub version:
> 
> ```dart
> dependencies:
>   fluent_ui:
>     git: https://github.com/bdlukaa/fluent_ui.git
> ```

Thanks for your reply, look forward to the new version!

**Comment by bdlukaa** [comment]
Published version 3.0.0. https://pub.dev/packages/fluent_ui/changelog
